src/sentry/utils/safe.py

```diff
@@ -115,21 +115,41 @@ def get_path(data: PathSearchable, *path, **kwargs):
     filtered with the given callback. Alternatively, pass ``True`` as filter to
     only filter ``None`` values.
     """
+    logger = logging.getLogger(__name__)
+    should_log = kwargs.pop("should_log", False)
```

Comment:
Nit: You could define this in the function definition like `data: PathSearchable, *path, should_log=False, **kwargs`

I'm not sure why we do all this popping here, might just be an artifact from python 2